sql/telemetry/feature_usage/query.sql

```diff
@@ -0,0 +1,114 @@
+SELECT
+  events_t.client_id AS clients_id,
+  events_t.submission_date AS submission_date,
+  COALESCE(protection_report, 0) AS protection_report,
+  COALESCE(pinned_tab, 0) AS pinned_tab,
+  COALESCE(default_browser, 0) AS default_browser,
+  COALESCE(many_uri, 0) AS many_uri,
+  COALESCE(pip, 0) AS pip,
+  COALESCE(filled_password, 0) AS filled_password,
+  COALESCE(remember_password, 0) AS remember_password,
+  COALESCE(sync_configured, 0) AS sync_configured
+FROM
+  (
+    SELECT
+      client_id,
+      submission_date,
+      IF(event_method = 'show' AND event_object = 'protection_report', 1, 0) AS protection_report,
+      IF(event_category = 'pictureinpicture' AND event_method = 'create', 1, 0) AS pip,
```

Comment:
IIUC, the intention is that we want one row in the final table per client_id per day. If so, each one of these subqueries should `GROUP BY submission_date, client_id` and each of these fields would be an aggregation like:

```
LOGICAL_OR(event_category = 'pictureinpicture' AND event_method = 'create') AS pip,
```

For cost and query efficiency, each of these features should be encoded as a boolean rather than as an integer (1 bytes vs. 8 bytes).

Reply:
Good point.  Will update.